### PR TITLE
[bugfix] analyzer conf doesn't accept primitives

### PR DESCRIPTION
**Pull request by marmoure**
currently per docs the analyzer config should accept primitive types as `int` & `char`
and this was not the case 
when using `int` as param type in the analyzer config it gets converted to `Integer` then checks against the existing `int` constructor which yields `no constructor found` error
this fixes it 
thanks to @adamretter for providing the fix code

### CODE REVIEW

File: extensions/indexes/lucene/src/main/java/org/exist/indexing/lucene/AnalyzerConfig.java
Comment (line-o) [inline extensions/indexes/lucene/src/main/java/org/exist/indexing/lucene/AnalyzerConfig.java:532]: Was this space added intentionally?

File: extensions/indexes/lucene/src/main/java/org/exist/indexing/lucene/AnalyzerConfig.java
Comment (line-o) [inline extensions/indexes/lucene/src/main/java/org/exist/indexing/lucene/AnalyzerConfig.java:616]: please remove surplus blank lines

**Comment by line-o** [comment]
This PR adds two pom.xml files that I have seen in my local checkouts as well. I wonder what these are and if they should be added or rathre ignored. If they should be added, this should happen in a separate commit. A separate PR even.

**Review by line-o — APPROVED** [review]
Other than the formatting this looks good to me.

**Comment by line-o** [comment]
Well, I would prefer the two pom files to be removed. And I would like to see the commits squashed before merging.

**Comment by marmoure** [comment]
> Well, I would prefer the two pom files to be removed. And I would like to see the commits squashed before merging.

oh yes sorry about those i added them by mistake

**Comment by marmoure** [comment]
@line-o  it should be good now

**Review by adamretter — COMMENTED** [review]
LGTM

**Review by dizzzz — APPROVED** [review]

**Comment by line-o** [comment]
We have two approvals. Should we wait until 6th of December before merging this (5 days of grace period)?

**Comment by dizzzz** [comment]
no, it is OK for me.

**Pull request merged**
